app-editors/vis/vis-0.4.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
```

Comment:
It's generally considered a good idea to test the ebuilds you're submitting because this one will certainly fail hard.